WireUI/Sources/WireIndividualToTeamMigrationUI/Components/PageContainer.swift

```diff
@@ -53,12 +53,14 @@ struct PageContainer<Content: View>: View {
                     Spacer(minLength: 36)
                     content
                 }
+                .padding(.horizontal, 16)
+                // Ensure there is a minimum of 16 points space to the bottom of the screen.
+                .padding(.bottom, max(16 - proxy.safeAreaInsets.bottom, 0))
```

Comment:
Here we make it so that the "Continue" button is 16 points from the bottom of the screen on an iPhone SE, and flush with the Botton of the safe area on larger devices